command.c

```diff
@@ -2743,11 +2745,56 @@ static void perform_external_cmd(int call, char *ext_cmd)
       memmove(cmd_args + 1, cmd_args, alen);
       cmd_args[0] = ' ';
       }
+    exefile = fopen(full_cmd,"rb");
+    if (exefile) {
+      /* from https://github.com/dosemu2/comcom32/issues/59#issuecomment-1179566783 */
+      unsigned char exebuffer[256] = { 0 };
+      unsigned is_mz_exe = 0;
+      fread(exebuffer, 1, 256, exefile);
+      if (exebuffer[0] == 'M' && exebuffer[1] == 'Z')
+        is_mz_exe = 1;
+      if (exebuffer[0] == 'Z' && exebuffer[1] == 'M')
+        is_mz_exe = 1;
+      if (is_mz_exe && exebuffer[16] == 128 && exebuffer[17] == 0
+        && (exebuffer[20] == 16 || exebuffer[20] == 18) && exebuffer[21] == 0) {
+        unsigned headersize = (exebuffer[8] + exebuffer[9] * 256UL) * 16UL;
+        short codesegment = exebuffer[22] + exebuffer[23] * 256UL;
+        unsigned checkoffset = headersize + ((int)codesegment * 16UL);
+        unsigned char entrybuffer[18] = { 0 };
+        fseek(exefile, checkoffset, SEEK_SET);
+        fread(entrybuffer, 1, 18, exefile);
+        if (entrybuffer[exebuffer[20] - 2UL] == 'R'
+          && entrybuffer[exebuffer[20] - 1UL] == 'B') {
+          do_auto_loadfix = 1;
+        }
+      }
+           if (is_mz_exe && !memcmp(&exebuffer[30], "PKLITE", 6))
```

Comment:
Bad indentation?

Reply:
That was intentional to make it easier to see the different spellings (capitalisation) of the "PKLITE" / "PKlite" strings.

Reply:
I guess in that case you had
to move the "PKLITE" string only,
not the entire Cish string.

Reply:
Allright.